test/metabase/lib/field/resolution_test.cljc

```diff
@@ -718,44 +718,6 @@
                    {:display-name "Sum of Total"}]
                   (lib/returned-columns query))))))))
 
-;;; adapted from [[metabase.queries.api.card-test/model-card-test-2]]
-(deftest ^:parallel preserve-model-metadata-test
```

Comment:
This is reworked in `master`, but relies on a lot of changes that I'm not backporting here. Just deleting for now to unblock release since it's not really important (we have e2e coverage for this anyway)